src/plugins/vis_type_markdown/public/types.ts

```diff
@@ -21,12 +21,12 @@ import { Style } from 'src/plugins/expressions/public';
 
 export interface Arguments {
   markdown: string;
-  font: Style;
-  openLinksInNewTab: boolean;
+  font?: Style;
+  openLinksInNewTab?: boolean;
```

Comment:
These should be optional as default values are provided in the function definition